projects/storefrontstyles/scss/components/cart/_cart-item.scss

```diff
@@ -106,11 +106,11 @@
 
   .cx-quantity {
     display: flex;
-    justify-content: center;
     align-items: center;
 
-    @include media-breakpoint-down(sm) {
-      justify-content: flex-start;
+    // Remove this wrapper for 5.0
+    @include media-breakpoint-up(md) {
```

Comment:
This change would need to be wrapped in `@include forVersion(4.2)` and wrap the code you replace in `@include forVersion(4.1)`